src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -159,10 +174,10 @@ public String toString() {
         return clients.asUnmodifiableObservableList();
     }
 
-    //TODO: For storing JSON notes
+    //TODO: add client notes also. NOTE: THIS ONLY RETURNS COUNTRY NOTES FOR NOW.
```

Comment:
alright will follow up on this once this PR is merged. my next task is delete command (at the least) before touching storage 👍